### DISCUSSION TITLE
FP 1.12.2_08: Message "locking of the map XXX failed. Acton Save as aborted

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1916] [Help]
The following message appears when I close FP (with that map still open):
![image](https://github.com/freeplane/freeplane/assets/62674157/b2a12102-7ece-4783-a03f-0f278b063086)
What I have to do to avoid this message?
Please note that the same map file in another FP installation doesn't create any message.

**Comment by dpolivaev** [comment]
See #1918
